crates/network/src/native/client.rs

```diff
@@ -119,33 +110,32 @@ impl ElementComponent for GameClientView {
                 control_tx.send(Control::Disconnect).ok();
             }
         });
+        let gpu2 = Arc::clone(&gpu);
```

Comment:
Why clone here and not in the scope of the places it's used? It wasn't immediately obvious to me where/why you'd be cloning this.